pkg/resourceinterpreter/defaultinterpreter/aggregatestatus.go

```diff
@@ -320,3 +321,51 @@ func aggregatePodStatus(object *unstructured.Unstructured, aggregatedStatusItems
 	pod.Status = *newStatus
 	return helper.ToUnstructured(pod)
 }
+
+func aggregatePersistentVolumeClaimStatus(object *unstructured.Unstructured, aggregatedStatusItems []workv1alpha2.AggregatedStatusItem) (*unstructured.Unstructured, error) {
+	pvc, err := helper.ConvertToPersistentVolumeClaim(object)
+	if err != nil {
+		return nil, err
+	}
+
+	newStatus := &corev1.PersistentVolumeClaimStatus{}
+	boundFlag := true
+	lostFlag := false
```

Comment:
1. New a PersistentVolumeClaimPhase initialized with `Bound`.
2. In for-range loop:
        If `Lost`, set it to Lost and break;
        If not `Bound`, set it.
3. Assign `newStatus.Phase`.